src/containers/App/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import HomepageHeader from "../../components/HomepageHeader/HomepageHeader.js";
+import PageLayout from "../PageLayout/PageLayout.js";
+import ErrorBoundary from "../ErrorBoundary/ErrorBoundary.js";
+
+export default function App() {
+  return (
+    <>
+      <ErrorBoundary>
+        <PageLayout>
+          <HomepageHeader />
+        </PageLayout>
+      </ErrorBoundary>
+    </>
```

Comment:
I know that is your task to use React Fragment, but try to use it when you really need it, like here:
[MovieList](https://github.com/zakhar98/react-gmp/pull/2/files#diff-5887ccd04b1e2255c870f39f9e527c3ae60a5b9216de602bf025562bccb4e6f0)

Reply:
You are right, fragment is redundant here